GeorgianMemenator/Scenes/CatTagsList/CatTagsListViewModel.swift

```diff
@@ -10,5 +10,12 @@ import SwiftUI
 import Foundation
 
 class CatTagsListViewModel: ObservableObject {
+    @Published var tags: [CatTagDetail] = [
+        CatTagDetail(uniqueID: 0, name: "Sphynx cat"),
+        CatTagDetail(uniqueID: 1, name: "Munchkin cat"),
+        CatTagDetail(uniqueID: 2, name: "Bengal cat")
+        ]
```

Comment:
Please format it properly, the `]` should not be here. CMD+A and then CTRL+I will format entire file